packages/cloud/deployment_details/services.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext } from 'react';
+
+export interface Services {
```

Comment:
über nit

```suggestion
export interface DeploymentDetailsServices {
```

or

```suggestion
export interface DeploymentDetailsContextValue {
```

Reply:
The main reason I use `Services` is so it's easier to copy it over when creating new pacakges 😊 But aggreed, `DeploymentDetailsContextValue` is more precise.